oauth2client/flask_util.py

```diff
@@ -331,6 +333,20 @@ def _make_flow(self, return_url=None, **kwargs):
             redirect_uri=url_for('oauth2.callback', _external=True),
             **kw)
 
+        flow_key = 'google_oauth2_flow_{0}'.format(csrf_token)
+        session[flow_key] = pickle.dumps(flow)
+
+        return flow
+
+    def _get_flow_for_token(self, csrf_token):
+        flow_pickle = session.get(
+            'google_oauth2_flow_{0}'.format(csrf_token), None)
```

Comment:
Actually I'm seeing them at class scope rather than instance scope. Please place them at module scope. It's extraordinarily rare for private constants to ever belong at any other scope, and this isn't one of those cases.